src/types.rs

```diff
@@ -820,27 +950,24 @@ impl Term {
             _ => None,
         }
     }
-    /// Compute the [alpha equivalence] for two terms.
+    /// Compute the [alpha equivalence] for two `Term`s.
     ///
     /// [alpha equivalence]: https://en.wikipedia.org/wiki/Lambda_calculus#Alpha_equivalence
     /// # Examples
     ///
     /// ```
-    /// # use term_rewriting::{Signature, parse_term,Term};
+    /// # use term_rewriting::{Signature, parse_term, Term};
     /// let mut sig = Signature::default();
-    /// let a = sig.new_op(2, Some(".".to_string()));
-    /// let s = sig.new_op(0, Some("S".to_string()));
-    /// let k = sig.new_op(0, Some("K".to_string()));
     ///
     /// let t = parse_term(&mut sig, "S K y_ z_").expect("parse of S K y_ z_");
     /// let t2 = parse_term(&mut sig, "S K y_ z_").expect("parse of S K y_ z_");
     /// let t3 = parse_term(&mut sig, "S K y_").expect("parse of S K y_");
     ///
-    /// let alpha_1 = Term::alpha(&t,&t2);
-    /// let alpha_2 = Term::alpha(&t,&t3);
-    /// let alpha_3 = Term::alpha(&t2,&t3);
-    /// assert_ne!(alpha_1,alpha_2);
-    /// assert_eq!(alpha_2,alpha_3);
+    /// let alpha_1 = Term::alpha(&t, &t2);
+    /// let alpha_2 = Term::alpha(&t, &t3);
+    /// let alpha_3 = Term::alpha(&t2, &t3);
+    /// assert_ne!(alpha_1, alpha_2);
+    /// assert_eq!(alpha_2, alpha_3);
```

Comment:
Hmm... what do you think of revising to run just two alpha comparisons (one that succeeds and one that fails) but showing what their outputs are?